packages/web-components/src/components/content-block-headlines/__stories__/README.stories.mdx

```diff
@@ -0,0 +1,118 @@
+import { Props, Description } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import markdownContents from '../../../../docs/markdown-contents.md';
+
+# `<dds-content-block-headlines>`
+
+> 💡 Check our
+> [CodeSandbox](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/components/content-block-headlines)
+> example implementation.
+
+[![Edit @carbon/ibmdotcom-web-components](https://codesandbox.io/static/img/play-codesandbox.svg)](https://githubbox.com/carbon-design-system/carbon-for-ibm-dotcom/tree/master/packages/web-components/examples/codesandbox/components/content-block-headlines)
+
+## Getting started
+
+Here's a quick example to get you started.
+
+##### JS
+
+```javascript
+import '@carbon/ibmdotcom-web-components/es/components/content-block-headlines/content-block-headlines.js';
+import '@carbon/ibmdotcom-web-components/es/components/content-block-headlines/content-block-headlines-item.js';
+import '@carbon/ibmdotcom-web-components/es/components/content-item/content-item-copy';
+```
+
+### HTML
+
+```html
+<dds-content-block-headlines>
+  <dds-content-block-heading>Aliquam condimentum<</dds-content-block-heading>
+  <dds-content-block-copy slot="copy"
+    >Lorem ipsum dolor sit amet, _consectetur_ adipiscing elit. Aenean et
+    ultricies est. Mauris iaculis eget dolor nec hendrerit. Phasellus at elit
+    sollicitudin.</dds-content-block-copy
+  >
+  <dds-content-block-headlines-item slot="content">
+    <dds-content-item-headline slot="headline">25%</dds-content-item-headline>
+    <dds-content-item-copy slot="copy"
+      >Modernize mission-critical applications and infrastructure in a hybrid
+      multicloud environment up to 25 percent faster.</dds-content-item-copy
+    >
+    <dds-link-list slot="cta" type="vertical">
+      <dds-link-list-item-cta
+        icon-placement="right"
+        href="https://www.ibm.com"
+        cta-type="local"
+      >
+        Link text
+      </dds-link-list-item-cta>
+    </dds-link-list>
+  </dds-content-block-headlines-item>
+  <dds-content-block-headlines-item slot="content">
+    <dds-content-item-headline slot="headline">25%</dds-content-item-headline>
```

Comment:
Couldn't find an implementation of `<dds-content-item-headline>` - Was this meant for another tag (component)? Or I'm missing something...?

Reply:
Got it – created a component for this. Thanks!

Reply:
Cool thanks @kennylam! Do you mean to push this change soon...?